README.md

```diff
@@ -5,27 +5,39 @@ After cloning this repo locally, install your desired addons:
     $ minishift addons install minishift-addons/knative-istio
     $ minishift addons install minishift-addons/knative-build
     $ minishift addons install minishift-addons/knative-serving
-    $ minishift addons install minishift-addons/knative-eventing
 
-All of the addons use the `KNATIVE_VERSION` environment variable to
+All of the above addons use the `KNATIVE_VERSION` environment variable to
 indicate which release is to be installed. The default (and probably
 minimum supported) is `v0.2.1`.
 
     $ minishift config set addon-env KNATIVE_VERSION=v0.2.1
 
 Knative components require istio, and the `knative-istio` addon
 contained in this repo applies the manifest published in official
-upstream knative repos.
+upstream Knative repos.
 
     $ minishift addons apply knative-istio
 
-Once that completes, install the knative resources you desire:
+Once that completes, install the Knative resources you desire:
 
     $ minishift addons apply knative-build
     $ minishift addons apply knative-serving
+
+## Knative Eventing
+
+The `knative-eventing` addon uses the `KNATIVE_EVENTING_VERSION` environment variable to
+indicate which release is to be installed. The default (and probably
+minimum supported) is `v0.2.0` in the case of *Knative Eventing*.
+
+    $ minishift addons install minishift-addons/knative-eventing
+
+Once that completes, install the Knative Eventing resources:
+
     $ minishift addons apply knative-eventing
```

Comment:
I was really hoping to avoid having a separate version env var for each addon, but maybe it's inevitable. What would you say to account for the lack of a 0.2.1 for eventing by suggesting this instead?
```suggestion
    $ minishift addons apply --addon-env KNATIVE_VERSION=v0.2.0 knative-eventing
```